### DISCUSSION TITLE
mathematical expressions  of the full conditional posterior distributions

### DISCUSSION THREAD

**Discussion by BahgatN** [pymc-labs/pymc-marketing#604] [Q&A] [answered]
I'm currently seeking the mathematical expressions for the full conditional posterior distributions, particularly focusing on the adstock rate and the logistic saturation parameter. Could you please direct me to any relevant documentation or articles? I'm primarily interested in understanding the statistical aspects rather than the coding implementation. Despite my search through the PyMC Marketing API documentation, I haven't been able to find what I'm looking for. Any help would be greatly appreciated.

**Comment by williambdean** [comment]
Hi @BahgatN 
I am trying to understand the question. 
The posteriors of the model will depend on the priors which are user specified. In general, I don't suspect there be a closed form distribution for the posteriors and that is why we rely on sampling methods like NUTS

**Comment by ColtAllen** [comment, marked as answer]
Mathematical expressions for the conditional posteriors do exist for the `CLV` Pareto/NBD and BG/BB models, but these models have strong underlying assumptions about how the customer data is distributed. I'm not aware of MMM having any such assumptions, in which case we must use Bayesian inference for the posteriors as  @wd60622 mentioned.
